common/src/test/scala/uk/ac/wellcome/test/utils/SQSLocal.scala

```diff
@@ -16,12 +24,26 @@ trait SQSLocal extends Suite with BeforeAndAfterEach {
       new EndpointConfiguration(s"http://localhost:9324", "localhost"))
     .build()
 
-  val idMinterQueue = "es_id_minter_queue"
-  val idMinterQueueUrl = sqsClient.createQueue(idMinterQueue).getQueueUrl
+  def queueName: String
+
+  // Use eventually to allow some time for the local SQS to start up.
+  // If it is not started all suites using this will crash at start up time.
+  val queueUrl = eventually {
+    sqsClient.createQueue(queueName).getQueueUrl
+  }
+
+  // Setting 1 second timeout for tests, to be able to test message deletion
```

Comment:
I’m not sure this follows. Couldn’t we test message deletion just as easily with a 15-second (I think that’s the default?) timeout by just waiting a bit longer? Or does the VisibilityTimeout default to infinity with our SQS local container?